packages/loader/container-loader/src/test/serializedStateManager.spec.ts

```diff
@@ -71,6 +69,10 @@ class MockStorageAdapter implements ISerializedStateManagerDocumentStorageServic
 			stringToBuffer(`{"minimumSequenceNumber" : 0, "sequenceNumber": 0}`, "utf8"),
 		);
 	}
+	public async updateGroupIdSnapshots(): Promise<void> {}
+	public get loadedGroupIdSnapshots(): Record<string, ISnapshot> {
+		return {};
+	}
 
 	public async getSnapshot(
```

Comment:
I believe adding some unit tests is a good idea. Implementing this method and write some here could be a way to do that.